### ISSUE TITLE
The game will crashes when I open a Gunbox's GUI

### ISSUE THREAD

**Issue by Nitric-acid-CN** [Unknown025/Flans-Mod-Plus#10]
The game will crashes when I open a Gunbox's GUI
The Crash Report:
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 20-9-12 下午9:51
Description: Rendering screen

java.lang.NullPointerException: Rendering screen
	at com.flansmod.client.gui.GuiGunBox.func_146979_b(GuiGunBox.java:79)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:119)
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1455)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at com.flansmod.client.gui.GuiGunBox.func_146979_b(GuiGunBox.java:79)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:119)
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1455)

-- Screen render details --
Details:
	Screen name: com.flansmod.client.gui.GuiGunBox
	Mouse location: Scaled: (286, 97). Absolute: (573, 524)
	Screen size: Scaled: (641, 360). Absolute: (1281, 720). Scale factor of 2
```

**Comment by Unknown025** [comment]
Thanks for reporting this. Do you know which version of Flan's Mod you are using?

**Comment by Nitric-acid-CN** [comment]
> Thanks for reporting this. Do you know which version of Flan's Mod you are using?

I'm using the lastest version 1.39.
It seems that there was this problem in past versions.

**Comment by GoldSloth** [comment]
Might be fixed by v1.40, could you try please?

**Comment by Unknown025** [comment]
I'm going to assume that this has been fixed, but feel free to reopen if that's not the case.
